ontology/yaml/resources/HVAC/entity_types/FCU.yaml

```diff
@@ -50,6 +50,27 @@ FCU_DFSS_CSP_DXZC:
   - CSP
   - DXZC
 
+FCU_DXRC_RMM:
+  description: "Typical DX FCU with cooling-only (return control) with run mode and failed alarm ."
+  is_canonical: true
+  opt_uses:
+  - failed_alarm
```

Comment:
is this the entire device is failed, or the discharge fan is failed?

Reply:
Entire device not only fan.